content/applications/hr/recruitment/offer_job_positions.rst

```diff
@@ -0,0 +1,147 @@
+     :guilabel:`Internal link` icon that appears to the right of the drop-down arrow. Make any
+     desired changes, then click :guilabel:`Save & Close`.
+
+- :guilabel:`Job Position`: the name of the :guilabel:`Job Position` being offered to the applicant.
+  The selections available in the drop-down menu correspond to the job position configured on the
+  main *Recruitment* dashboard.
+- :guilabel:`Job Title`: the position being offered to the applicant.
+- :guilabel:`Department`: the :guilabel:`Department` the job position falls under.
+- :guilabel:`Contract Start Date`: the date the contract takes effect. The default date is the
+  current date. To modify the date, click on the displayed date to reveal a calendar pop-up window.
+  Navigate to the desired month, then click the day to select the date.
+- :guilabel:`Yearly Cost`: the annual salary being offered.
+- :guilabel:`Link Expiration Date`: the number of days the job offer is valid. The default
+  expiration date is `30` days. Modify the expiration date, if desired.
+
+Send offer
+----------
+
+Once the :guilabel:`Generate a Simulation Link` pop-up window is correct, click :guilabel:`Send By
+Email` to reveal an email pop-up window.
+
+.. important::
+   If the applicant does not have an email address listed on their applicant card, a warning appears
+   in a red box at the bottom of the :guilabel:`Generate a Simulation Link` pop-up window, stating:
+   :guilabel:`The applicant does not have a valid email set. The Offer Link won't be able to be
+   completed.` Click :guilabel:`Discard`, then enter an email on the applicant's card. Once an email
+   is entered, click the :guilabel:`Generate Offer` button, and the email pop-up window loads again.
+
+The default :guilabel:`Recruitment: Your Salary Package` email template is used (set in the
+:guilabel:`Load template` field), and the :guilabel:`Recipients`, :guilabel:`Subject`, and email
+body are pre-populated based on the email template.
+
+If any attachments need to be added, click the :icon:`fa-paperclip` :guilabel:`Attachments` button,
+and a file explorer window appears. Navigate to the desired file, then click :guilabel:`Open` to
+attach it to the email. The attachment loads, and is listed above the :icon:`fa-paperclip`
+:guilabel:`Attachments` button.
+
+Once the email is ready to send, click :guilabel:`Send`. The email pop-up window closes, and an
+:guilabel:`Offers` smart button appears at the top of the applicant's card.
+
+.. note::
+   To send an offer, ensure the *Sign* application is installed. This is necessary so the offer can
+   be sent to the applicant by the recruiter, and they can sign the offer. The applicant does
+   **not** need any software installed to sign the offer.
+
+.. image:: offer_job_positions/send-offer.png
+   :align: center
+   :alt: Send an email to the applicant with a link to the offered salary.
+
+Configure your package
```

Comment:
```suggestion
Configure package
```
I understand this is a button, but it's my understanding that we should avoid using "you"/"your" whenever possible